layersvt/device_simulation.cpp

```diff
@@ -1319,6 +1483,12 @@ VKAPI_ATTR void VKAPI_CALL DestroyInstance(VkInstance instance, const VkAllocati
     }
 }
 
+// Utility struct to check sType of pNext chain structures.
```

Comment:
Look up `VkBaseInStructure` and `VkBaseOutStructure`. I'm pretty sure it's better to use those instead of creating your own type.

Reply:
Will make this change.  It's easy and cleaner.